index.bs

```diff
@@ -3218,7 +3308,7 @@ definitions.
         };
     </pre>
 
-    Note that the [{{Constructor}}] and
+    Note that [=constructor methods=] and
```

Comment:
Do we still need the prose mention here now that the grammar enforces it?

Reply:
It's just a note; I think it's okay to keep.